### ISSUE TITLE
Accessibility: Issue with Screen Reader and Sign In page

### ISSUE THREAD

**Issue by clepore** [infor-design/enterprise#421]
When reviewing our sign in page for accessibility using the MS Windows Screen Reader, I've noticed a problem with Chrome where it does not read the "user" field. The same occurs in the standard sign-in example with the email field.

http://usalvlhlpool1.infor.com/4.4.0-rc/components/signin/example-index

If you tab between the email and password fields in IE11, it reads out both labels. If you do the same in Chrome, it reads the password but when you tab back to the user field, it says "hidden".

An additional chrome issue is with icon buttons on the sign in that we have. In IE11 it reads out the audible label, in Chrome it just says button.

Also, in both browsers, it does not say anything about a field being required. In Chrome it reads out the "asterix". I was expecting different verbal feedback for required and not required fields.

Formerly: SOHO-7473

**Comment by janahintal** [comment]
QA Passed. Will move to Done. Thanks!
